core/lib/dal/src/events_dal.rs

```diff
@@ -84,7 +84,7 @@ impl EventsDal<'_, '_> {
                 write_str!(
                     &mut buffer,
                     r"\\x{topic0:x}|\\x{topic1:x}|\\x{topic2:x}|\\x{topic3:x}|",
-                    topic0 = EventTopic(event.indexed_topics.get(0)),
+                    topic0 = EventTopic(event.indexed_topics.first()),
                     topic1 = EventTopic(event.indexed_topics.get(1)),
                     topic2 = EventTopic(event.indexed_topics.get(2)),
                     topic3 = EventTopic(event.indexed_topics.get(3))
```

Comment:
Damn clippy, you're ruining all the aesthetics!

Reply:
yeah...